### ISSUE TITLE
[Help/Wiki] How to make action work?

### ISSUE THREAD

**Issue by talmachani** [OpenPoGo/OpenPoGoWeb#164]
I have enabled pokemon actions in userdata.js , i get 404 on the polling (i'm guessing that it),
I thought that maybe I need to enable web socket property in config, but still nothing happens.

This is a great feature!!

How do I make it work?
BTW bot is on branch dev

**Comment by BobbyWibowo** [comment]
Did you use OpenPoGoBot? The web interface is transiting to support OpenPoGoBot only. Well, at least the action feature is supposed to only work with OpenPoGoBot since it requires WebSocket or something.

**Comment by talmachani** [comment]
im using https://github.com/PokemonGoF/PokemonGo-Bot
they actually look pretty similar

**Comment by BobbyWibowo** [comment]
Yeah, you can't use the actions with PokemonGo-Bot. It doesn't have complete WebSocket implementation yet, although I'm sure they're currently developing it in their dev branch. For now, I'm sure you'll have to use OpenPoGoBot if you really want to use the actions menu.
